AdminServer/setup.py

```diff
@@ -38,6 +39,8 @@
   packages=['appscale',
             'appscale.admin',
             'appscale.admin.instance_manager'],
+  package_data={'': ['*.json']},
+  include_package_data=True,
```

Comment:
The only documentation I was able to find for this argument was in reference to a `Manifest.in` file. Does it have a different behavior here?

Reply:
I think it is the same, i.e. that the package data should be distributed.